observelib/observe/src/main/java/org/ballerinalang/observe/nativeimpl/ObserveNativeImplConstants.java

```diff
@@ -33,7 +33,7 @@
     private ObserveNativeImplConstants() {
     }
 
-    public static final Module OBSERVE_PACKAGE_ID = new Module(BALLERINA_ORG.getValue(), "observe", "0.9.0");
+    public static final Module OBSERVE_PACKAGE_ID = new Module(BALLERINA_BUILTIN_PKG_PREFIX, "observe", "0.9.0");
```

Comment:
May be we can get rid of this contant and use io.ballerina.runtime.Environment.getCurrentModule.

Reply:
https://github.com/ballerina-platform/module-ballerina-io/pull/49/files

Reply:
Fixed in e692962